content/posts/tau/index.md

```diff
@@ -18,12 +18,92 @@ summary: "A customize osu! ruleset with a circle surrounding a paddle and some n
 - GitHub Repositories : [Altenhh/tau](https://github.com/Altenhh/tau/)
 - Discord : [tau](https://discord.com/invite/7Y8GXAa)
 
-Tau is a custom ruleset with a circle surrounding a paddle and some notes. Based on idea from [pizzazip's Tau](https://deadlysprinklez.itch.io/tau). The ruleset is developed by Laica.
+Tau is a custom ruleset with a circle surrounding a paddle and some notes.
+
+ *The idea is based on [pizzazip's Tau](https://deadlysprinklez.itch.io/tau). The ruleset is developed by [Laica](https://github.com/Altenhh).*
 
 # gameplay
 
 Tau's gameplay revolves around a paddle which is moved around a circle with the mouse. In the circle, there are different types of notes trying to escape it. The player has to move the paddle to block these notes, then hit them using keyboard keys.
-You can see all note types in [Tau gameplay note types wiki](https://github.com/Altenhh/tau/wiki/Note-types).
+
+The gameplay playfield looks like the following:
+
+![Playfield](Images/Playfield.png)
+
+This is an empty playfield, the parts it contains are the following:
+
+![Cursor](Images/Cursor.png) ![Paddle](Images/Paddle.png)
+
+ This is the cursor   /       This is the paddle
+  1.  The Cursor
+    This is your mouse, it shows you where on the screen your mouse currently is.
+    
+  *The closer the cursor is to the middle of the screen the faster your paddle moves, the further away the cursor is from the middle of the playfield the slower your paddle moves (Because the circle you move your mouse in becomes smaller) keep this in mind while playing to make it easier to play*
+  2. The Paddle 
+    You move it by moving your mouse and cursor, you use it to hit the notes and not let them escape your grasp.
+    *You should always move it to the place the next note you are gonna hit is heading and keep it there until the note intersects the paddle and you press the button required for that specific note.*
```

Comment:
![Screen Shot 2564-05-03 at 03 31 37](https://user-images.githubusercontent.com/68165621/116826806-13d92300-abc0-11eb-9161-4dd863fe1b98.png)
Change linespace
```suggestion
  1.  The Cursor
  
  This is your mouse, it shows you where on the screen your mouse currently is.
    
  *The closer the cursor is to the middle of the screen the faster your paddle moves, the further away the cursor is from the middle of the playfield the slower your paddle moves (Because the circle you move your mouse in becomes smaller) keep this in mind while playing to make it easier to play*
  
  2. The Paddle 
  
    You move it by moving your mouse and cursor, you use it to hit the notes and not let them escape your grasp.
    
    *You should always move it to the place the next note you are gonna hit is heading and keep it there until the note intersects the paddle and you press the button required for that specific note.*
```